### DISCUSSION TITLE
Default sort descending in data tab

### DISCUSSION THREAD

**Discussion by BCPaulPaul** [pawelsalawa/sqlitestudio#4880] [General]
Is there a setting to change the default sort to descending when viewing a table data tab? 90% of the time this is what I want.

**Comment by pawelsalawa** [comment]
Well, sorting is not only about ascending/descending, but also about which column (or columns) to sort by.

By default SQLiteStudio does not sort anything. The order is determined purely by SQLite (and as far as I remember it is insertion order).

How about creating views like `SELECT * FROM my_table ORDER BY my_column DESC`? SQLiteStudio will let you edit data through such view.
